skimage/measure/_structural_similarity.py

```diff
@@ -3,36 +3,137 @@
 __all__ = ['structural_similarity']
 
 import numpy as np
+from scipy.ndimage.filters import uniform_filter, convolve1d
 
 from ..util.dtype import dtype_range
-from ..util.shape import view_as_windows
 
 
-def structural_similarity(X, Y, win_size=7,
-                          gradient=False, dynamic_range=None):
+def gaussian_filter2(X, sigma=1.5, size=11):
```

Comment:
Double check: can't this be done using ndimage's gaussian filter implementation?